map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -330,8 +330,8 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
 
         viewModel.selectedArticObject
                 .withLatestFrom(viewModel.displayMode) { selected, mapMode -> selected to mapMode }
-                .filterFlatMap({ (_, mapMode) -> mapMode is MapDisplayMode.CurrentFloor }, { (selected) -> selected })
-                .subscribeBy { selected ->
+                .filterFlatMap({(_, mapMode) -> mapMode is MapDisplayMode.CurrentFloor },{(selectedObject, mapMode) -> selectedObject to mapMode as MapDisplayMode.CurrentFloor})
+                .subscribeBy { (selected, _) ->
```

Comment:
Does this change have any functional side-effect?

I presume it's just to make the code more readable.

Reply:
no undoing this change i forgot about that little portion my first stab at adding the map analytics was wrong

Reply:
Well, it does make it more readable. Feel free to add it back in a separate PR.